server/sandbox_network.go

```diff
@@ -54,51 +54,60 @@ func (s *Server) networkStart(sb *sandbox.Sandbox) (podIP string, result cnitype
 		return
 	}
 
-	podIP = strings.Split(network.IPs[0].Address.String(), "/")[0]
-
-	if len(sb.PortMappings()) > 0 {
-		ip := net.ParseIP(podIP)
-		if ip == nil {
-			err = fmt.Errorf("failed to get valid ip address for sandbox %s(%s)", sb.Name(), sb.ID())
-			return
-		}
-
-		err = s.hostportManager.Add(sb.ID(), &hostport.PodPortMapping{
-			Name:         sb.Name(),
-			PortMappings: sb.PortMappings(),
-			IP:           ip,
-			HostNetwork:  false,
-		}, "lo")
-		if err != nil {
-			err = fmt.Errorf("failed to add hostport mapping for sandbox %s(%s): %v", sb.Name(), sb.ID(), err)
-			return
+	for _, podIPConfig := range network.IPs {
+		podIP := strings.Split(podIPConfig.Address.String(), "/")[0]
+
+		if len(sb.PortMappings()) > 0 {
+			ip := net.ParseIP(podIP)
+			if ip == nil {
+				err = fmt.Errorf("failed to get valid ip address for sandbox %s(%s)", sb.Name(), sb.ID())
+				return
+			}
+
+			err = s.hostportManager.Add(sb.ID(), &hostport.PodPortMapping{
+				Name:         sb.Name(),
+				PortMappings: sb.PortMappings(),
+				IP:           ip,
+				HostNetwork:  false,
+			}, "lo")
+			if err != nil {
+				err = fmt.Errorf("failed to add hostport mapping for sandbox %s(%s): %v", sb.Name(), sb.ID(), err)
```

Comment:
Do we need to add the `ip` field to this message?   Perhaps  `... hostport mapping using IP %v for sandbox ...`